src/ctap/hid/mod.rs

```diff
@@ -62,27 +63,8 @@ pub struct Message {
     pub payload: Vec<u8>,
 }
 
-pub struct CtapHid {
-    assembler: MessageAssembler,
-    // The specification (version 20190130) only requires unique CIDs ; the allocation algorithm is
-    // vendor specific.
-    // We allocate them incrementally, that is all `cid` such that 1 <= cid <= allocated_cids are
-    // allocated.
-    // In packets, the ID encoding is Big Endian to match what is used throughout CTAP (with the
-    // u32::to/from_be_bytes methods).
-    allocated_cids: usize,
-    pub wink_permission: TimedPermission,
-}
-
-#[allow(dead_code)]
-pub enum KeepaliveStatus {
-    Processing,
-    UpNeeded,
-}
-
-#[allow(dead_code)]
-// TODO(kaczmarczyck) disable the warning in the end
-impl CtapHid {
+pub struct CtapHidConsts;
```

Comment:
I wonder why this extra class is necessary?

Reply:
Because CtapHid depends on Generic C: Clock. However, some of the Consts are used in places where adding this C generic is not necessary. If we don't extract these consts out, then there will be more generics added to more classes and functions.

Reply:
Now I removed the generics I can put them back. But rethinking on it, maybe it is not a bad idea to have them separate. WDYT?

Reply:
What benefit do you have in mind? Otherwise, a smaller diff in this PR is welcome :)

I guess any future modifications of the nature would benefit from the separation, not sure if it's worth it, if it's just that?